First proposal for linking MathML challenges in MathML best practices

gregoriopellegrino opened the pull request.

avneeshsingh commented:
The content is good. I think we should not use respec in the challenges document. This should look little different from best practices. Best practices is more formal document and respec provides it the feel of formal document, while challenges is a more frequently updated descriptive document. So, it can look less formal (without using respec).

gregoriopellegrino commented:
Now it's plain HTML (Is there a basic CSS template to make it more appealing?).
I added a reference to the main document.
Perhaps a reference to DAISY is missing (logo or just plain text).

gregoriopellegrino commented:
Here I am:

- I retrieved the template for the Reading apps UX requirements document
- I fixed the header; for now, I've hidden the information about the editors and the document history
- I added styles to the CSS to manage the notes
